startup.py

```diff
@@ -21,6 +21,6 @@
     elif choice == '2':
         dbmanager.stop()
     elif choice == '3':
-        initdb.main("","")
+        initdb.main()
```

Comment:
Whoa whats going on? @kevintaing86 don't pull @dlowrey's branch. That could seriously mess things up when we bring the 2 current Pull Requests into `master`